tests/e2e/helpers/notifications.ts

```diff
@@ -0,0 +1,43 @@
+import { join } from 'path';
+import * as os from 'os';
+import { NotificationParameters } from '../pageObjects/notification-page';
+
+const workingDirectory = process.env.APP_FOLDER_ABSOLUTE_PATH
+    || (join(os.homedir(), process.env.APP_FOLDER_NAME || '.redisinsight-v2'));
+const dbPath = `${workingDirectory}/redisinsight.db`;
+console.log(`dbPath: ${dbPath}`);
+
+const sqlite3 = require('sqlite3').verbose();
+
+export function deleteAllNotificationsFromDB(): void {
+    const db = new sqlite3.Database(dbPath);
+    db.run('DELETE from notification', function(err: { message: string }) {
+        if (err) {
+            return console.log(`error during notification deletion: ${err.message}`);
+        }
+    });
+    db.close();
+}
+
+export function insertNotificationInDB(notifications: NotificationParameters[]): void {
```

Comment:
Please add details and parameters for each function.

Reply:
Added comments to all the functions